### PR TITLE
New changes done by malgosia

### PR DESCRIPTION
**Pull request by malgosias**
have a look at the changes and please accept them

### CODE REVIEW

**Comment by jlgalache** [comment]
Thanks Malgosia, that was a really useful addition to the project!

File: readme.md [hunk 1]
```diff
@@ -11,3 +11,6 @@ This new line is exciting because I'm writing it via the Github interface.
 Changes made on local file to clash with Github changes.
 
 Making a change on Github to purposely interfere with my local version
+
+I added this new line to your readme.md
```
Comment (malgosias) [inline readme.md:15]: are these changes needed?

Now JL can write here too!
